membership-attribute-service/app/Global.scala

```diff
@@ -0,0 +1,34 @@
+import controllers.{NoCache, Cached}
+import filters.AddEC2InstanceHeader
+import models.{MembershipAttributes, ApiError, ApiErrors, ApiResponse}
+import play.api.Logger
+import play.api.mvc.{RequestHeader, Result, WithFilters}
+import play.filters.csrf._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object Global extends WithFilters(CSRFFilter(), AddEC2InstanceHeader) {
+
+  private val logger = Logger(this.getClass)
```

Comment:
That trait looks like its mainly for access logs and metrics rather than application logging. So its use is probably only appropriate in controllers?